### ISSUE TITLE
ember test broken with latest stable blueprint

### ISSUE THREAD

**Issue by toranb** [glimmerjs/glimmer.js#68]
I found the command `ember test` blows up using both node 8.9 and node 7.5 with both ember-cli 2.16.2 and 2.17.0-beta.2

To reproduce this simply spin up a new glimmer app like so

`ember new demo -b @glimmer/blueprint`

Then run `ember test` and you will see an exception complaining that `@glimmer/test-helpers` cannot find `setApp` in the dist/modules/es2017/index.js file

![screen shot 2017-11-11 at 8 30 26 am](https://user-images.githubusercontent.com/147411/32690355-115fd8aa-c6bb-11e7-9b5a-b5597053b495.png)

Taking a quick look at the [commits](https://github.com/glimmerjs/glimmer-test-helpers/commits/master) for glimmer-test-helpers and it seems like not much has changed. When I dove into the node_modules folder itself `@glimmer/test-helpers/dist/modules/es2017/index.js` this is what I found

```js
export { assertIsElement, assertNodeTagName, assertNodeProperty, equalInnerHTML, equalHTML, equalTokens, generateSnapshot, equalSnapshots, normalizeInnerHTML, isCheckedInputHTML, getTextContent, strip, stripTight, trimLines, blockStack, assertSerializedInElement } from './lib/helpers';
export { TestDynamicScope, equalsElement, inspectHooks, regex, classes } from './lib/environment';
export * from './lib/render-test';
export * from './lib/suites';
export { HelperReference } from './lib/environment/helper';
export { default as TestMacros } from './lib/environment/macros';
export { default as AbstractTestEnvironment } from './lib/environment/environment';
export * from './lib/environment/modifier';
export { default as LazyTestEnvironment, default as TestEnvironment } from './lib/environment/modes/lazy/environment';
export { NodeLazyRenderDelegate, NodeEagerRenderDelegate } from './lib/environment/modes/ssr/environment';
export { default as EagerRenderDelegate } from './lib/environment/modes/eager/render-delegate';
export { default as LazyRenderDelegate } from './lib/environment/modes/lazy/render-delegate';
export { debugRehydration } from './lib/environment/modes/rehydration/debug-builder';
export { default as EagerTestEnvironment } from './lib/environment/modes/eager/environment';
export * from './lib/environment/components';
```

Below is the broken example app for reference but honestly you should be able to create this yourself using the very latest stable blueprint with any recent ember-cli

https://github.com/toranb/busted-glimmer

**Comment by rwjblue** [comment]
Can you `npm ls @glimmer/test-helpers`?

**Comment by rwjblue** [comment]
The issue is that `@glimmer/test-helpers` was accidentally published from glimmer-vm (and your yarn.lock shows that you are using the affected version 0.30.1). This was fixed originally in https://github.com/glimmerjs/glimmer-vm/pull/563 but that work was lost in some refactors. @chadhietala discovered that things were broken again and marked the glimmer-vm package as private again in https://github.com/glimmerjs/glimmer-vm/pull/740.

Ultimately, we need to publish a new version of @glimmer/test-helpers from https://github.com/glimmerjs/glimmer-test-helpers so that apps pick this version up instead...

**Comment by rwjblue** [comment]
I published `@glimmer/test-helpers@0.30.2` in https://github.com/glimmerjs/glimmer-test-helpers/commit/dcdf068495b05355968e285fb36b4adc7609ebb1 to allieviate the issue...

**Comment by toranb** [comment]
Thanks @rwjblue
